src/main/solidity/com/ampnet/identityservice/contract/Issuer.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IIssuer.sol";
+
+contract Issuer is IIssuer {
```

Comment:
Is there a possibility to use just interface instead of the whole implementation?

Reply:
It is, and I suggest doing it with an interface.

Reply:
There is no method  ``` function approveWallet(address _wallet)```  defined in the interface. Check it at https://github.com/AMPnet/tokenizer-prototype/blob/master/contracts/issuer/IIssuer.sol. It seems more reasonable to copy/paste the exact code as is from the smart contract then define my own interface. I don't see the Issuer.sol as such an overhead.